py/desispec/qa/qalib.py

```diff
@@ -738,8 +738,36 @@ def SNRFit(frame,night,camera,expid,params,fidboundary=None,
                             minchi2=chi2
                             fita=guess[0]
                             fitb=guess[1]
+                #- Increase granualarity of 'a' by a factor of 10
+                for c in range(100):
+                    for d in range(100):
```

Comment:
OK for now, but for the record, this brute force search looks slow, and without outlier rejection I would expect it to have convergence problems on real data.  I would expect that one of the routines in scipy.optimize could be much faster, enabling a few iterations of outlier rejection. Consider that for a future PR.